src/screens/CreatePost/index.js

```diff
@@ -670,7 +667,7 @@ const CreatePost = () => {
   }, 500);
 
   const onChangeText = (text) => {
-    setMessage(text);
+    setMessage(debounced(text));
     handleTagUser(text);
   };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on implementing debouncing on the `onChangeText` function. This will prevent unnecessary re-renders and improve performance by limiting the rate at which the `setMessage` function is invoked as the user types. Just make sure that the `debounced` function is correctly implemented and tested.


<!-- This is an auto-generated comment by OSS CodeRabbit -->